### DISCUSSION TITLE
Feat: Openshift CLI (oc) integration

### DISCUSSION THREAD

**Discussion by Metal-spoon** [aws/codewhisperer-command-line-discussions#115] [Feature request]
### Checks

- [X] I could not find an existing setting that solved my issue
- [X] I searched [the discussions in this repo](https://github.com/aws/codewhisperer-command-line-discussions/discussions?q=) and there are no duplicates of my feature request

### Description:
Redhat provides a CLI based on kubectl for their openshift platform called OC.
Getting CLI completions for this would be very useful for the people that use it.

Given that it's based on kubectl, which already has a CLI integration, this shouldn't be that much of a stretch.

Source code for OC is available here: [https://github.com/openshift/oc](https://github.com/openshift/oc)

**Comment by grant0417** [comment]
All of our completions are opensource and this can be contributed at https://github.com/withfig/autocomplete.
